workshop content of example career path. (#145)
CaptainSifff/paper_teaching-learning-RSE: example_career_path.md

# Example Career path for an RSE

+ For now this is developed in a separate file, since we don't know where it is going to end up (e.g. section in current paper, appendix of current paper, new paper)
+ based on [notes from un-deRSE23](https://pad.gwdg.de/BrHmFT7QQHKJmq4i3hn-Fg#)
+ Taking hints (aka plagiarism) from [this F1000 article](https://f1000research.com/articles/9-295/v2)

## Intro

Meet Kay, who currently studies researchology in a Bachelor program student in researchonomy at University of Orithena (UofO).
We will follow Kay’s fictional career to illustrate how education, job-experience and a career in academic institutions could play together to produce a successful RSE.
In Kay’s world, some of the measures this paper proposes have already been implemented.

## Bachelor

> Through a program like DaLiJe or The Missing Semester, Kay learns about using computational tools to support the sophisticated statistical analysis typical for researchology.

Reviewer: We could use https://www.dataliteracy.uni-jena.de/zertifikatsprogramm as a link.

> Through a program like DaLiJe or The Missing Semester, Kay learns about using computational tools to support the sophisticated statistical analysis typical for researchology.

Reviewer: For "The missing semester" there is a git repo (https://github.com/missing-semester/missing-semester) and a website (https://missing.csail.mit.edu/), of which the git repo might be the more useful. The material is CC-BY-NS-SA, but I could not find a specific text how or what exactly they want to have cited.

> Through a program like DaLiJe or The Missing Semester, Kay learns about using computational tools to support the sophisticated statistical analysis typical for researchology.

Author: I think we already have it in our bigraphy. the key is athalye2023.

He uses those tools for creation and automation when compiling his Bachelor thesis (generating plots with matplotlib, maybe even CI for automatic building) 
and takes pride in a fully open and reproducible bachelor thesis.

## Master

Kay ponders whether to continue with computational researchology, which her Bachelor supervisor is responsible for, or enroll in a generic RSE Master’s program.
Researchers in computational researchology need to acquire a large part of the general RSE know-how presented

> in this paper and specialize in Quantum-Accelerated Bayesian Optimization methods.

Reviewer 2: love it, QABO!

> However, Kay decides to go the more generic route of a dedicated RSE programme because she wants to continue in academia,

Reviewer 3: ```suggestion
However, Kay decides to go the more generic route of a dedicated RSE programme because she wants to continue in academia
```

but doesn't like the idea to become stuck with one research topic.
She also observed that the immediate satisfaction gained by helping colleagues from her research group with tricky technical
problems makes her happier than the subdued sense of achievement from having a research paper accepted long after she has written it.

+ The generic RSE Master’s consists has a core of RSE topics with various electives for specialization
+ Kai chooses digital archaeology

> + Develops pipeline for reconstruction of 3D models from under ground sonar data, to simplify the process for archaeologists (Reproducibility, Big Data, ML?)

Reviewer 2: sonar is in the sea ;-). this is pretty standard stuff. maybe there is some ML. It could be big data. Developing pipelines is definitely an improvement

Reviewer: I guess this would then be Ground Penetrating Radar (GPR) or Radio Echo Sounding (RES).

> The project requires her to apply the project management skills that are being taught as part of the core RSE curriculum.

Reviewer 3: ```suggestion
The project management skills that are being taught as part of the core RSE curriculum really help her to not get lost in this project.
```

Apart from working with the researchers in her archaeology group she has to work with members of the central RSE department to help her with the pipelines. 

> She also has to liaise with the central IT department to organise storage for the large data sets.

Reviewer 3: ```suggestion
She also has to connect with the central IT department to organise storage for the large data sets.
```

Towards the end of the programme she visits her first RSE conference.
Kay has now completed the RSE programme and has reached Junior RSE level.

## Junior RSE

Kay finds a position in the central RSE department at her university with a competitive IT salary. 

> Although the contract is temporary there is a good chance that it will lead to an open ended position.

Reviewer 3: ```suggestion
Although the contract is temporary there is a good chance that it will lead to a permanent position.
```

+ Teaches the basics to other researchers
+ Teaches a seminar in the RSE Master’s program
+ Works there for two years

## Senior RSE

+ Takes a position as a RSE responsible within a new research group at another university
+ Takes responsibility for RSE related parts of research of a large inter-organizational project (Verbundprojekt) as part of this new position

> + Senior RSE skills ...

Reviewer 2: we'll need to expand this a little

> The new position involves taking responsibility for the RSE related aspects of a large inter-organisational project (Verbundprojekt). 

Reviewer 3: ```suggestion
The new position involves taking responsibility for the RSE related aspects of a large inter-organisational project. 
```
That should probably go before merging
